php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -666,5 +674,60 @@ class NDB_Menu_Filter extends NDB_Menu
         $html = $smarty->fetch("menu_$this->menu.tpl");
         return $html;
     }
+
+    /**
+     * Controls access to a module's SQL tables, exporting a CSV file of the table
+     * to user. This script should ensure that only files relative to module's path
+     * specified are accessible.
+     * @return void
+     * @access private
+     */
+    function _getCSVData()
+    {
+        $list['data']    = $this->_getFullList();
+        $list['headers'] = $this->headers;
+
+        $csv_data = Utility::arrayToCSV($list);
+        header("HTTP/1.1 200 OK");
+        print $csv_data;
+        exit(2);
```

Comment:
I think it would make more sense to have the above 3 lines where this is called, and have _getCSVData() return the data, otherwise this function name is misleading.. I wouldn't expect a function called "get" to print and exit.